src/utils/geo.js

```diff
@@ -76,3 +76,32 @@ export function scrambleLocation(center, radiusInMeters) {
 export function kmToMiles(km) {
   return km / KM_TO_MILES;
 }
+
+/**
+ * Returns the reverse geocoded results for the given latitude and logitude values using Google
+ * Maps api. This assumes that the Google Maps API is already loaded and available globally as
+ * window.google.maps.
+ * @param {Number} latitude Latitude
+ * @param {Number} longitude Latitude
+ */
+export async function reverseGeocode(latitude, longitude) {
+  if (!window.google?.maps?.Geocoder) {
+    // eslint-disable-next-line no-console
+    console.error('Google Maps API not found.');
+    return null;
```

Comment:
We may want to pass this case along to analytics or Sentry so that we know how often this is happening. It can happen in the case of the client blocking these types of requests using a browser extensions like Ghostery

Reply:
Good point.  I did Sentry integration after this change so didn't account for it here.  There are a few other Google Maps places where Sentry errors would be helpful.